### PR TITLE
style<settings>: Set maximum width for buttons to improve internationalization

### LINKED ISSUE #5968: Set maximum width for buttons to improve internationalization
I still think that CSS grid is a proper solution, but in the meantime, let's set a maximum width, no wrap, no overflow with ellipsis truncation of buttons and provide the full copy in the title for hover.

internationalize

### PR DESCRIPTION
**Pull request by hritvi**
fixes #5968

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eda01cab13b386b85e88c91c4292bd853ef7b9a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e242cd2ff6213d82d8539e91cd4259c7abbfb031
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/styles/modules/_settings.scss [hunk 1]
```diff
@@ -759,8 +759,10 @@ section.modal-panel {
         /*minimum width required for the button to look good without occupying too much space*/
         /*is also the default computed width on desktop screen*/
         min-width: 100px;
+        overflow: hidden;
```
Comment (vladikoff) [inline app/styles/modules/_settings.scss]: @hritvi do you also need to use `text-overflow: ellipsis;`?

Example: https://github.com/mozilla/notes/blob/19632423890f6de87cb5f37ec8fe571a2bc0479b/src/sidebar/static/scss/footer.scss#L43

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eda01cab13b386b85e88c91c4292bd853ef7b9a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: e242cd2ff6213d82d8539e91cd4259c7abbfb031
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8eda01cab13b386b85e88c91c4292bd853ef7b9a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: e242cd2ff6213d82d8539e91cd4259c7abbfb031
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by hritvi** [comment]
Please review @vladikoff! Thanks :)

File: app/scripts/templates/settings/emails.mustache [hunk 2]
```diff
@@ -30,7 +30,7 @@
                                 </div>
                                 <div class="settings-button-group">
                                     {{#verified}}
-                                        <button class="settings-button secondary set-primary" data-id="{{email}}">
+                                        <button class="settings-button secondary set-primary" data-id="{{email}}" title ="{{#t}}Make primary{{/t}}">
```
Comment (vladikoff) [inline app/scripts/templates/settings/emails.mustache]: remove the space between `title =`

File: app/styles/modules/_settings.scss [hunk 3]
```diff
@@ -759,8 +759,11 @@ section.modal-panel {
...
         text-align: center;
-        width: 20%;
+        text-overflow: ellipsis;
```
Comment (vladikoff) [inline app/styles/modules/_settings.scss]: can we apply the same styles to try and fix these buttons: 
image

**Pull request closed without merging**